src/components/Header.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const Header = () => {
+  const headerStyle = {
+    padding: '20px 0',
+    lineHeight: '1.5em',
+  };
+  return (
+    <header style={headerStyle}>
+      <h1
+        style={{
+          fontSize: '6rem',
+          fontWeight: '600',
+          marginBottom: '2rem',
+          lineHeight: '1em',
+          color: '#ececec',
+          textTransform: 'lowercase',
+          textAlign: 'center',
+        }}
```

Comment:
To keep your code simple and clean, I think this styling can be done inside the `module.css`  file, just as you did that of the `TodoItem` component. This will keep your code clean and modular to manage in case of future features 😃